Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -65,7 +65,12 @@ public static Task ExecuteOnUIThreadAsync(CoreApplicationView viewToExecuteOn, F
                 viewToExecuteOn,
                 async () =>
                 {
-                    await function().ConfigureAwait(false);
+                    var task = function();
+                    if (task == null)
+                    {
+                        throw new InvalidOperationException("The Task returned by function cannot be null.");
+                    }
+                    await task.ConfigureAwait(false);
```

Comment:
Why await the result? Just return the result of the method and remove the async keyword to the anonymous method.